solidstart-tanstackquery-tailwind-modules/msw/data.ts

```diff
@@ -0,0 +1,71 @@
+export const readMeResponse = {
+  data: {
+    repository: {
+      readme: {
+        text: 'text',
+      }
+    }
+  }
+};
+
+export const repoInforResponse = {
+  data: {
+    repository: {
+      defaultBranchRef: {
+        name: 'MockBranch'
+      },
+      isPrivate: true,
+      forkCount: true,
+      description: 'This is a description',
+      homepageUrl: 'https://www.google.com',
+      stargazerCount: 1,
+      watchers: {
+        totalCount: 2
+      },
+      issues: {
+        totalCount: 2
+      },
+      pullRequests: {
+        totalCount: 2
+      },
+      topics: {
+        nodes: ['topic1', 'topic2', 'topic3']
+      },
+      owner: {
+        orgName: 'tested'
+      }
+    }
+  }
+}
+
+export const repoTreeResponse = {
```

Comment:
why is this data here? It's not part of this ticket

Reply:
I shouldn't have done this.